plugins/janus_videoroom.c

```diff
@@ -9970,7 +9975,7 @@ static void *janus_videoroom_handler(void *data)
 												 JANUS_SDP_OA_DONE);
 				/* Is this room recorded, or are we recording this publisher already? */
 				janus_mutex_lock(&participant->rec_mutex);
-				if (videoroom->record || participant->recording_active)
```

Comment:
I would set the  videoroom->record in the code of "create" and "init" video room.

Instead of:
videoroom->record = **janus_is_true(record->value);**
Set:
videoroom->record = **true**

and not changing the logic itself.